src/jlgr.jl

```diff
@@ -1,10 +1,71 @@
 module jlgr
 
 import GR
+# These methods are extended in jlgr
+import GR: hexbin, contour, contourf, surface
```

Comment:
One extra thing that needed to be done was to import the `hexbin`, `contour`, `contourf`, and `surface` methods in `jlgr` so that they could be extended.